conf/cqlshrc.sample

```diff
@@ -109,6 +112,9 @@ port = 9042
 
 
 ;[ssl]
+;; Version of TLS to be used. Required to be set for cassandra version > 3.11.8
```

Comment:
@neshkeev I do not think this is necessary to add, there is this later in `ssl` section:

````
; this is effectively ignored from 4.1 included as TLS protocol is auto-negotiated and will
; be removed in the next major version of Cassandra, possible values were TLSv1, TLSv1_1 or TLSv1_2
;version =
````

Reply:
I removed it

Reply:
https://issues.apache.org/jira/browse/CASSANDRA-17452